ext/mysqli/tests/bind_fetch/smallint_types.phpt

```diff
@@ -1,38 +1,35 @@
 --TEST--
-mysqli fetch short values
+mysqli fetch small/short integer values
 --EXTENSIONS--
 mysqli
 --SKIPIF--
 <?php
-require_once('skipifconnectfailure.inc');
+require_once dirname(__DIR__) . "/test_setup/test_helpers.inc";
+mysqli_check_skip_test();
 ?>
 --FILE--
 <?php
-    require_once("connect.inc");
+require_once dirname(__DIR__) . "/test_setup/test_helpers.inc";
 
-    /*** test mysqli_connect 127.0.0.1 ***/
-    $link = my_mysqli_connect($host, $user, $passwd, $db, $port, $socket);
+    $link = default_mysqli_connect();
 
     if (!mysqli_query($link, "SET sql_mode=''"))
         printf("[001] [%d] %s\n", mysqli_errno($link), mysqli_error($link));
 
-    if (!mysqli_query($link, "DROP TABLE IF EXISTS test_bind_fetch"))
-        printf("[002] [%d] %s\n", mysqli_errno($link), mysqli_error($link));
-
-    $rc = mysqli_query($link, "CREATE TABLE test_bind_fetch(c1 smallint unsigned,
+    $rc = mysqli_query($link, "CREATE TABLE test_bind_fetch_integers_small(c1 smallint unsigned,
                                                      c2 smallint unsigned,
                                                      c3 smallint,
                                                      c4 smallint,
                                                      c5 smallint,
                                                      c6 smallint unsigned,
-                                                     c7 smallint) ENGINE=" . $engine);
+                                                     c7 smallint) ENGINE=" . get_default_db_engine());
```

Comment:
Does the engine need to be actually specified? Because again we are not very consistent in that regards...

Reply:
I have no idea. To my knowledge, the engine doesn't matter here. I am not entirely sure what this test actually tests.